content/docs/guides/certificates.md

```diff
@@ -37,6 +37,41 @@ data:
 
 For details and code where this is used see [osm-controller.go](https://github.com/openservicemesh/osm/blob/release-v0.9/cmd/osm-controller/osm-controller.go#L182-L183).
 
+### Rotate the Root Certificate (Tresor)
+
+The self-signed root certificate, which is created via the Tresor package within OSM, will expire in a decade. To rotate the root cert, the following steps should be followed: 
+
+1. Delete the `osm-ca-bundle` certificate in the osm namesapce
+   ```console
+   kubectl delete secret osm-ca-bundle -n osm-system
+   ```
+2. Restart the `osm-controller` pod(s) in the osm-namespace
+   ```console
+   kubectl delete pod <osm-controller-pod> -n osm-system
+   ```
+
```

Comment:
We should document that all control plane components - osm-bootstrap, osm-controller, and osm-injector need to be restarted so they pick up the new root CA certificate similar to install.